channels/do.m3u

```diff
@@ -37,8 +37,12 @@ https://ss2.domint.net:3200/gdm_str/gdmtv/playlist.m3u8
 https://cloudflare.streamgato.us:3352/live/ghtelevisionhdlive.m3u8
 #EXTINF:-1 tvg-id="HermanasMirabalTV.do" tvg-country="DO" tvg-language="" tvg-logo="https://emisoradominicana.net/television/wp-content/uploads/2015/06/hermanas-mirabal-tv.jpg" group-title="",Hermanas Mirabal TV (720p) [Not 24/7]
 https://cm.hostlagarto.com:4445/Hmtv/hmtv.myStream/playlist.m3u8
-#EXTINF:-1 tvg-id="HilandoFino.do" tvg-country="DO" tvg-language="Spanish" tvg-logo="http://www.hilandofinotv.com/wp-content/uploads/2017/04/cropped-logo-tv-3.png" group-title="",Hilando Fino (720p) [Offline]
+#EXTINF:-1 tvg-id="HilandoFino.do" tvg-country="DO" tvg-language="Spanish" tvg-logo="http://www.hilandofinotv.com/wp-content/uploads/2017/04/cropped-logo-tv-3.png" group-title="",Hilando Fino (1080p) [Not 24/7]
 https://4k-server-mia.2cdn.eu/HilandoFinoTV/ngrp:HilandoFinoTV_all/playlist.m3u8
```

Comment:
Drops connection for me atm

Reply:
Interesting, last time I tried it worked. The other links also work ?

Reply:
They do

Reply:
![image](https://user-images.githubusercontent.com/30985701/136029778-516e4254-adeb-4c32-96a4-c07697542ea3.png)

Link works for me

Reply:
Maybe is geo-blocked

Reply:
I'll try with a russian IP.

Reply:
> 
> 
> > @sguinetti Cuando usted podrias, probas este link y digame si esta functionando.
> 
> Hola. No me funciona para mí. Pienso que está geobloqueado aunque para ser un canal latinoamericano, lo dudo. Entonces, ¿será problema técnico?

Es una otra problema, ese link functiona en el sitio y en Firefox con el extencion, pero no functiona en VLC, tengo un error 503

Reply:
https://primary-out.iptv-global.net/HilandoFinoTV/ngrp:HilandoFinoTV_all/chunklist_w1513336512_b3128000.m3u8

Reply:
That link works, also. (ffplay, Firefox, VLC)